Newsletters/SecretDecks to Occultish.md

```diff
@@ -1,9 +1,14 @@
 ## Intro
 Calling all creators! SecretDecks® looks to you if you’ve ever reinterpreted the Tarot and made your own deck... or perhaps you feel inspired to do so? 
-Are you an artist? Build a new global fanbase, make a living off your digital art without any middlemen, as well as own & manage your copyrights!
-What if you're not an aritst? Join us anyway and find out how you can help us to help each other and what you can get in return! 
+
+### Are you an artist? Do you like Tarot?
+Build a new global fanbase, make a living off your art without any middlemen and through the medium of your choice, as well as own & manage your copyrights!
+
+### What if you're not an aritst? 
```

Comment:
### What if you're not an artist ?

Reply:
What am I missing here? It looks like you changed nothing?

Reply:
think it was just a typo correction for "artist"

Reply:
Yes - just "artist" spelling @DanM3rcurius